### PR TITLE
Portfolio: Milestone6: Accessibility

### PR DESCRIPTION
**Pull request by aliabbani**
## 
In this milestone we:

- checked all the points 
- Page titles: we edited the page title.
- Image text alternatives: we edited all the alt attributes.
- Text headings: we follow the rule of the page heading.
- Color contrast: no issues found
- Resize: no issues found
- Interaction: no issues were found
- Moving content: no issues found
- Multimedia: no issues found
- The basic structure of the page: we also edited the form section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -282,25 +282,31 @@ <h2 class="skill-title">Skills</h2>
...
+      <label for="name">
+        <input type="text" name="name" id="name" maxlength="30" placeholder="yourname" required />
+      </label>
+      <label for="email">
+        <input type="email" name="email" id="email" placeholder="example@mail.com" required />
+      </label>
+      <label for="message">
+        <textarea name="message" id="message" rows="5" maxlength="500" placeholder="Write your message here" required>
 
       </textarea>
```
Comment (Emmyn5600) [inline index.html:301]: - [ ] Kindly, fix your contact form, it allows me to send empty message. please make it required.

File: index.html [hunk 2]
```diff
@@ -282,25 +282,31 @@ <h2 class="skill-title">Skills</h2>
...
+      </label>
+
       <div class="button-container">
         <button class="button" type="submit">Get In Touch</button>
       </div>
     </form>
 
+
     <div class="bottom-line"></div>
```
Comment (Emmyn5600) [inline index.html:310]: - [ ] Please double check the color contrast of the project. Use this [link](https://www.aditus.io/button-contrast-checker/) to check for the color contrast and fix it.

File: index.html [hunk 3]
```diff
@@ -282,25 +282,31 @@ <h2 class="skill-title">Skills</h2>
     </section>
   </main>
   <footer class="contact-sect">
-    <h1 class="header-text contact-header-text">contact me</h1>
+    <h2 class="header-text contact-header-text">contact me</h2>
     <p class="primary-text-about contact-primary-text-about">
       If you have an application you are interested in developing, a feature
       that you need built or a project that needs coding. I’d love to help
       with it
     </p>
```
Comment (Emmyn5600) [inline index.html:290]: - [ ] Some of the colors you've chosen for your portfolio currently don't meet WebAIM accessibility standards for contrast ratio. Please readjust your color palette by using a color contrast validator tool like this [one](https://color.a11y.com/%5C)  to assess contrast ratio.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: Changes Requested :recycle:

Hi @zieeco 

Good job so far!:clap::clap:
You have done well in implementing the project requirements :clap:, However, there are some issues that you still need to work on to go to the next project but you are almost there!:muscle:

##  Required Changes:writing_hand:

**_Check the comments under the review._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by carlos-ssh — APPROVED** [review]
Hi @zieeco, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**